slack/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.togglz</groupId>
+    <artifactId>togglz-project</artifactId>
+    <version>2.4.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>togglz-slack</artifactId>
+  <name>Togglz - Slack integration</name>
+  <description>Togglz - Slack integration</description>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>17.0</version>
```

Comment:
To be honest, I don't like having a dependency on Guava at all. Don't get me wrong, I love Guava. For my applications I use it very often. But I had major issues in the past where different 3rd party libraries depended on different older Guava versions which very totally incompatible. That's because the Guava guys remove deprecated functionality soon after the deprecation. And this can get you into Guava version hell. That's why I think that libraries (like Togglz) should not depend on Guava. What do you use Guava for?